### DISCUSSION TITLE
[Q&A] 我自己部署也想要PWA这个下载功能

### DISCUSSION THREAD

**Discussion by 5478-lfy** [lobehub/lobehub#1151] [Self Hosting] [answered]
![image](https://github.com/lobehub/lobe-chat/assets/92673213/2986ee4b-f7df-4a2a-9560-8fd6ce8286fa)
老师，方便告诉在哪里做修改或者要加什么东西吗？谢谢

**Comment by arvinxx** [comment]
自部署的不支持吗？应该支持的呀？

**Comment by 5478-lfy** [comment]
![image](https://github.com/lobehub/lobe-chat/assets/92673213/45aa22d7-9e8c-4efa-948c-7b94da3e8639)
没有，老师，我再重新部署一下看一下

**Comment by 5478-lfy** [comment]
![image](https://github.com/lobehub/lobe-chat/assets/92673213/52482f5a-6ae1-469f-a25f-e09629e0bf3b)
以本地方式打开是有的
但是如果用域名的方式打开是没有的，这个是为什么
![image](https://github.com/lobehub/lobe-chat/assets/92673213/a509c931-1f73-4210-8da3-90515e5dd71a)
@arvinxx

**Reply by arvinxx** [reply, marked as answer]
<img width="1155" alt="image" src="https://github.com/lobehub/lobe-chat/assets/28616219/aa60c996-698d-411d-a518-5c6c569bf8c7">

看看AI的回复，可能是因为你没有 https 吧
